tests/TestCase/ORM/ResultSetTest.php

```diff
@@ -328,7 +328,7 @@ public function testHasOneEagerLoaderLeavesEmptyAssocation()
     public function testFetchMissingDefaultAlias()
     {
         $comments = TableRegistry::get('Comments');
-        $query = $comments->find();
+        $query = $comments->find()->select(['Other__field' => 'test']);
```

Comment:
It relied on the results from the mocked statement. The new implementation requires fields to be present in the select clause